spec/factories/conversation.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+    factory :conversation do
+      recipient { 'Name' }
+      subject { 'MyString' }
+      body { 'MyText' }
+    end
+  end
```

Comment:
What are you using this factory for? 
The only model you are in need for is the user model, that you already have.